libs/apriltag/CMake/apriltagHelperFunctions.cmake

```diff
@@ -0,0 +1,28 @@
+# Helper Functions
+function(set_apriltag_lib_property_defaults target_name)
+    set_target_properties(${target_name} PROPERTIES SOVERSION 3 VERSION ${PROJECT_VERSION})
+    set_target_properties(${target_name} PROPERTIES DEBUG_POSTFIX "d")
+    set_target_properties(${target_name} PROPERTIES POSITION_INDEPENDENT_CODE ON)
+endfunction()
+
+function(set_apriltag_named_exports_only target_name)
+    set_target_properties(${target_name} PROPERTIES 
+        C_VISIBILITY_PRESET hidden 
+        CXX_VISIBILITY_PRESET hidden
+        VISIBILITY_INLINES_HIDDEN ON
+        WINDOWS_EXPORT_ALL_SYMBOLS OFF
+        # multiple libs sharing same auto-generated header so override default to use apriltag_EXPORTS
+        DEFINE_SYMBOL apriltag_EXPORTS
+    )
+endfunction()
+
+function(set_apriltag_export_all target_name)
```

Comment:
Can you add a 1~2 line comment to each of the three functions to explain what they are doing?